clippy_lints/src/loops.rs

```diff
@@ -66,6 +66,12 @@ declare_clippy_lint! {
     ///     println!("{}", vec[i]);
     /// }
     /// ```
+    /// Could be written as:
+    /// ```ignore
```

Comment:
Good examples should use `rust` instead of `ignore`.

Reply:
you are right. Best practices need to be respected :smiley: 
I will change the existing example as well ;)